legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,241 @@
 [[macros.html]]
-
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-      <title>Amalgamation Application</title>
-      <meta charset="UTF-8">
-      <meta name="author" content="BC Registries and Online Services">
-      [[common/style.html]]
-    </head>
-    <body>
-    </body>
+  <head>
+    <title>Amalgamation Application</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="BC Registries and Online Services">
+    [[common/style.html]]
+  </head>
+  <body>
+    <div class="header">
+      <table class="header-table" role="presentation">
+        <tr class="no-page-break">
+          <td>
+            [[logo.html]]
+          </td>
+          <td>
+            <div class="report-type">AMALGAMATION APPLICATION</div>
+            <div class="report-type-desc">BC Limited Company - Business Corporations Act</div>
+          </td>
+        </tr>
+      </table>
+      <div class="business-name-header">
+        <label class="lbl-business-name">{{business.legalName}}</label>
+      </div>
+    </div>
+    [[common/businessDetails.html]]
+    <div class="no-page-break">
+      <div class="section-header mt-5 pt-4">AMALGAMATION APPLICATION</div>
+    </div>
+
+    <!-- Name Request [[amalgamation/nameRequest.html]]-->
```

Comment:
I found I could not use [[amalgamation/nameRequest.html]], presumably because I am not inputting the path into a template. Definitely need assistance here as this file is much too long.

Reply:
Looks a lot better, thanks! 👍